### ISSUE TITLE
Inaccessible color contrast in HTML input color when Dark mode

### ISSUE THREAD

**Issue by bramaudi** [kiwibrowser/src.next#402]
![Screenshot_Kiwi_Browser_20210825-140059.png](https://user-images.githubusercontent.com/14371119/130756273-b6561d48-d2d5-49ba-afa8-5fbdc27141c5.png)

As we can see (hope you can) the text color on "Set" and "Cancel" button is blend with the black background, need a while until i realize where that buttons are 😅

**Comment by KaKi87** [comment]
Can you please share the website ?

**Comment by bramaudi** [comment]
Sorry i didn't check this issue for the long time, now i check it again after your reply and the problem was already gone, tested again with the same website & w/ dark mode as always (I'm not sure why it fixed but i have upgrade the app like ±2 times after do factory reset).

This is not related to specific website i guess since the issues is in HTML Color input UI.

![Screenshot_Kiwi_Browser_20210929-155912-picsay.png](https://user-images.githubusercontent.com/14371119/135239393-73b0dbdd-6f98-494d-a30e-78a13cbd0020.png)

**Comment by KaKi87** [comment]
Alright, thanks.
